packages/wasm-terminal/src/process/process.ts

```diff
@@ -155,11 +156,10 @@ export default class Process {
     length: number = stdinBuffer.byteLength,
     position?: number
   ) {
-    if (this.readStdinCounter > 0) {
-      this.readStdinCounter--;
+    if (this.readStdinCounter % 2 !== 0) {
```

Comment:
Oh yeah, let's remove this when we can. Can you open an issue for this? Or open a TODO? 🤔

Reply:
Done! https://github.com/wasmerio/wasmer-js/issues/160